external/fv3fit/tests/training/_regtest_outputs/test_train.test_train_default_model_on_identity[sklearn_random_forest].out

```diff
@@ -1,3 +1,3 @@
 ('var_in', '98a134e67c4a2b9a2da5d9744988191f')
 ('var_out', '98a134e67c4a2b9a2da5d9744988191f')
-('var_out', 'f92a68c6d69eb995dbf2375d3de9bc9f')
+('var_out', '848a1d247289e7ef7e698fdf4622a9c4')
```

Comment:
Do you have some idea of why this changed? It's OK for it to change, but ideally we know why.

Reply:
This is outdated, previously the regtest output changed because I had a commit that set the random states in the training function rather than at the top level in train.py.